daemon/src/model/cfd.rs

```diff
@@ -251,6 +254,17 @@ pub enum CfdState {
         cet_status: CetStatus,
     },
 
+    /// The collaborative close transaction was published but it not final yet
```

Comment:
:wrench: Little typo.

```suggestion
    /// The collaborative close transaction was published but is not final yet.
```